pallets/liquid-staking/src/lib.rs

```diff
@@ -1020,11 +1080,15 @@ pub mod pallet {
             );
 
             let ledger: StakingLedger<T::AccountId, BalanceOf<T>> =
-                Self::staking_ledgers(&derivative_index).ok_or(Error::<T>::NotBonded)?;
+                Self::staking_ledger(&derivative_index).ok_or(Error::<T>::NotBonded)?;
             ensure!(
                 ledger.unlocking.len() < MAX_UNLOCKING_CHUNKS,
                 Error::<T>::NoMoreChunks
             );
+            // ensure!(
+            //     ledger.active.saturating_sub(amount) >= T::MinNominatorBond::get(),
```

Comment:
same here, we still need `ensure` here, cannot fully trust Strategy

Reply:
will cancel code annotation now, let's make the Strategy more robust